rails_application/test/test_helper.rb

```diff
@@ -81,11 +81,16 @@ def register_customer(name)
   end
 
   def register_product(name, price, vat_rate)
+    add_available_vat_rate(vat_rate)
```

Comment:
Do you think this is fine?
If two products will be created with the same vat rate then this will attempt to create two vat rates with the same code. The second attempt will fail and return 422 response. This does not interfere with test logic but I'd like to know if making this call every time when we register product is fine.

The other option is to change every test using `register_product` so they also explicitly create vat rates.

Reply:
I am afraid that having such solution would lead to hacking. Also, it would make the method name invalid.

I'd prefer if the `add_available_vat_rate` is called just once. In fact, all we need is to have available rates defined before tests are run.

Reply:
Thanks 👍 I didn't know how to do it in a way that it is called only once but I added it to each test needing it (inside a setup block).